### PR TITLE
chore: Update templates for issues and PRs

### PR DESCRIPTION
**Pull request by sillyElvy**
## About

This pull request is about (X), which does (Y).

## Checklist

- [ ] I've ran `pre-commit` to format and lint the change(s) made.
- [ ] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [ ] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues).
  - (If existent):
- I've made this pull request for/as: (check all that apply)
  - [ ] Documentation
  - [ ] Breaking change
  - [ ] New feature/enhancement
  - [ ] Bugfix

### CODE REVIEW

**Review by Toricane — APPROVED** [review]
~~LGTM!~~

File: .github/ISSUE_TEMPLATE/report-bug.yml [hunk 1]
```diff
@@ -7,9 +7,9 @@ assignees:
   - EdVraz
   - Astrea49
   - Toricane
+  - Catalyst4222
 body:
   - type: markdown
-    id: thanks
```
Comment (Toricane) [inline .github/ISSUE_TEMPLATE/report-bug.yml:12]: wait a second, why was this removed?

File: .github/ISSUE_TEMPLATE/request-feature.yml [hunk 2]
```diff
@@ -7,9 +7,9 @@ assignees:
   - EdVraz
   - Astrea49
   - Toricane
+  - Catalyst4222
 body:
   - type: markdown
-    id: thanks
```
Comment (Toricane) [inline .github/ISSUE_TEMPLATE/request-feature.yml:12]: same with this

**Review by Toricane — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/request-feature.yml (hunk 2 above)
Comment (FayeDel) [inline .github/ISSUE_TEMPLATE/request-feature.yml:12]: Why is the ID removed here? (and in report-bug.yml as well)

Comment (Toricane) [reply .github/ISSUE_TEMPLATE/request-feature.yml:12]: lol ive asked the same, see above

File: .github/ISSUE_TEMPLATE/report-bug.yml (hunk 1 above)
Comment (sillyElvy) [reply .github/ISSUE_TEMPLATE/report-bug.yml:12]: Pycharm said it was invalid

Comment (Toricane) [reply .github/ISSUE_TEMPLATE/report-bug.yml:12]: did it say why?

Comment (sillyElvy) [reply .github/ISSUE_TEMPLATE/report-bug.yml:12]: "key `id` not allowed here"

[some github documentation link]

Comment (sillyElvy) [reply .github/ISSUE_TEMPLATE/report-bug.yml:12]: I'll check it again later

Comment (sillyElvy) [reply .github/ISSUE_TEMPLATE/report-bug.yml:12]: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

As you can see here markdown fields are not supposed to have an ID

**Review by Toricane — APPROVED** [review]
LGTM!
